### PR TITLE
feat: [PT-22] add Flipper component and tests

### PR DESCRIPTION
**Pull request by VladSolodjankin**
#### Description
Add Flipper component and add flipper to message menu button

| Details                     | Description                                      |
| --------------------------- | ------------------------------------------------ |
| JIRA ticket                 | https://marathon-health.atlassian.net/browse/PT- |
| Design source               | N/A                                              |
| Related PRs                 | N/A                                              |
| Documentation update needed | No                                               |
| QA sign-off by              | N/A                                              |

## Changelog

- 

## Checklist

- [X] [Commit message convention](https://kb.epam.com/display/MAHE/Code+Style) followed.
- [X] [Branch naming convention](https://kb.epam.com/display/MAHE/Code+Style) followed.
- [X] Performed self review & removed the debugging code (debugger, console log etc).
- [X] Updated/Added tests.
- [X] Added explanatory comment for tricky code explaining "why".
- [X] Tested on major browsers (Chrome, Firefox, Safari, Edge)
- [X] Manual testing or integration testing done.
- [X] Added strings via locale

## Screenshots / Animated GIFs

| Before | After |
| ------ | ----- |

### CODE REVIEW

File: src/modules/flippers/__tests__/Flipper.test.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+describe('Testing Flipper component', () => {
+  test('Flipper should show element', () => {
+    render(
+      <Flipper flipper={flipper}>
+        <div>show_test_flipper</div>
+      </Flipper>,
+      { preloadedState: flipperActive },
+    );
+
+    expect(screen.getByText(/show_test_flipper/i)).toBeInTheDocument();
```
Comment (dasarisamanth) [inline src/modules/flippers/__tests__/Flipper.test.jsx:26]: Please use data-testid if you can )

**Review by dasarisamanth — APPROVED** [review]
Please address the one comment I left, other than that everything else LGTM

File: src/modules/flippers/__tests__/Flipper.test.jsx (hunk 1 above)
Comment (VladSolodjankin) [reply src/modules/flippers/__tests__/Flipper.test.jsx:26]: no based on Guiding Principles it is better to use getByText

https://testing-library.com/docs/queries/about#priority

Comment (VladSolodjankin) [reply src/modules/flippers/__tests__/Flipper.test.jsx:26]: no based on Guiding Principles it is better to use getByText

https://testing-library.com/docs/queries/about#priority

**Review by EvgeniyKvasyuk — APPROVED** [review]

**Pull request merged**